### DISCUSSION TITLE
Logitech g9x front side button lowers screen brightness. App options do not affect button 4/5

### DISCUSSION THREAD

**Discussion by Fed-Marchesi** [noah-nuebling/mac-mouse-fix#258] [Questions] [answered]
Can I provide logs to help ensure this can be addressed?
Would be awesome to use this mouse more effectively.

**Comment by noah-nuebling** [comment, marked as answer]
Hi! Marking this as answered since I've moved the topic to #354. Please continue discussion of the issue there. Thanks!
